source/tools/applications.txt

```diff
@@ -25,11 +25,15 @@ projects supporting MongoDB.
 
 Applications Using MongoDB
 --------------------------
+- `Bussion <http://www.bussion.com>`_
+
+Create beautiful & visually appealing dashboards with drilldown and collabration capabilities. 
+Connect Bussion to any Mongo Db and create Widgets and Dashbords within minutes.
```

Comment:
s/Mongo Db/MongoDB/
s/Widgets/widgets/
s/dashbords/dashboards/ (note that you have spelled dashboards incorrectly.)